### PR TITLE
Gets rid of (most) horny things [DNM]

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
Gets rid of as much horny stuff as I could find. Unticks all lewd item / structure files, among many other things. I haven't tested this yet, so dear lord please do not merge this yet

Also, keeps pillows, collars, and feathers.

## How This Contributes To The Skyrat Roleplay Experience
Host request

## Changelog

:cl:
del: The Horny
/:cl:

### CODE REVIEW

**Comment by comradef191** [comment]
clearly we must speedmerge this-

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -270,6 +270,7 @@
 			"You hear a slap.")
 			target.dna?.species?.stop_wagging_tail(target)
 			return
+/* //ACULASTATION EDIT START - ASS SLAPPING BE GONE
```
Comment (PetMudstone) [inline code/modules/mob/living/carbon/carbon_defense.dm]: This section of commented out code caused errors while compiling on my end. Strangely, Zonespace states that he didn't have any trouble on his end. Would like another party to verify.

Comment (Zonespace27) [reply code/modules/mob/living/carbon/carbon_defense.dm]: CI passes, I have no idea why it's strange like this

**Comment by mopcat** [comment]
looks like you missed the erp section under game preferences, may wanna disable that since even turning that on seems to break stuff in the char creator among other obvious reasons

EDIT: there's also the "kinky collar" in the loadout menu, that could at least use a rename

**Comment by Nobelium-XVIII** [comment]
If I remember correctly, BYOND loads/compiles folders in order of name. You'll have to rename `modular_aculastation` to something like `modular_Z_aculastation` if you want it to overwrite any SR changes.

**Comment by Zonespace27** [comment]
> If I remember correctly, BYOND loads/compiles folders in order of name. You'll have to rename `modular_aculastation` to something like `modular_Z_aculastation` if you want it to overwrite any SR changes.

good catch, done

**Comment by comradef191** [comment]
Fullmerging because TGS is fucking stupid: A naked view option would be nice (as was on the ardent branch) for application of markings or w/e they were called

**Pull request merged**